CameraXBasic/app/src/main/java/com/android/example/cameraxbasic/fragments/CameraFragment.kt

```diff
@@ -488,6 +494,9 @@ class CameraFragment : Fragment() {
             framesPerSecond = 1.0 / ((timestampFirst - timestampLast) /
                     frameTimestamps.size.coerceAtLeast(1).toDouble()) * 1000.0
 
+            // Simulate long time analyze
+            Thread.sleep(1000)
+
             // Calculate the average luma no more often than every second
             if (frameTimestamps.first - lastAnalyzedTimestamp >= TimeUnit.SECONDS.toMillis(1)) {
```

Comment:
This is no longer necessary thanks to your change, remove these two lines and the corresponding closing brace :)

Reply:
For some reason GitHub won't mark the lines if they are not part of your change. I'm referring to lines 500-501.